nvflare/app_common/workflows/statistics_controller.py

```diff
@@ -295,3 +326,33 @@ def _prepare_inputs(self, metric_task: str, fl_ctx: FLContext) -> Shareable:
 
     def _get_result_cb(self, metric_task: str):
         return self.result_callback_fns[metric_task]
+
+    def _wait_for_all_results(
+        self, result_wait_timeout: int, request_client_size: int, client_metrics: dict, abort_signal=None
+    ) -> bool:
```

Comment:
args can be called just "timeout" and "metrics"?
"request_client_size" -> "num_of_results"/"size"?

Reply:
maybe want to add docstring/comment to explain this timeout is reset for each metric, it is not the overall timeout